### DISCUSSION TITLE
DCDAO Advocate

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#36] [Ideas]
# About DCDAO

The Darwinia Community DAO (DCDAO) is a community self-organized workgroup (WG) that aims to contribute to the growth and development of the Darwinia community. Sponsored by the Darwinia Treasury and Itering, the DCDAO welcomes donations and contributions from community members who share its vision and objectives.

# What is the DCDAO Advocate?

DCDAO is excited to extend its team members who will help shape the future of the organization. DCDAO advocate is aiming at injecting the new blood into DCDAO and Darwinia. Regardless of your background, whether you’re a content writer, designer, developer, trader, community moderator, or crypto enthusiast, we invite you to apply. We’re particularly interested in individuals who actively promote the DCDAO and create a positive environment in our social media channels,  we value and aim to reward! We encourage more members to interact with us and let more new members work with us.

# What does a DCDAO Advocate expect to do?

As a DCDAO Advocate, you have the flexibility to contribute in ways that align with your abilities, such as:

1. Creating a positive environment in Telegram chat, Twitter, or other social media platforms.
2. Spreading awareness about DCDAO through engaging content, including tweet threads, blogs, infographics, memes, and more.
3. Active engagement with DCDAO official social media posts and mentions of DCDAO.
4. Generating innovative ideas to benefit the DCDAO and community members.
5. Managing local communities and translating official news into your local language for sharing.
6. Attend online and offline cryptocurrency events and promote DCDAO's vision.
7. Initiating collaborations with other projects for potential partnerships.

# What will you get?

1. Early insights into Darwinia’s latest features & announcements.
2. The opportunity to provide valuable feedback on the latest Darwinia platform features.
3. The opportunity to be an official DCDAO  ambassador and participant in DAO’s governance.
4. 100$–300$ Monetary incentives in RING (reward amount based on advocate’s contributions).
5. Direct access to core team members for any Darwinia platform-related and DCDAO queries.
6. Exceptional efforts will be met with exceptional rewards.
# How to join it?
Click this link and fill it out with your information!
https://docs.google.com/forms/d/1f15Vj28xl3jWmAqZnAC6ggjiqFbTrfyl9TkfqQBnnSA/edit
Share your ideas in the comments area!

**Comment by JuneCribb** [comment]
Amazing!! How do I get in?

**Reply by Cindy884** [reply]
Hi June, you can contact me via discord.

**Comment by hackfisher** [comment]
Hi Cindy,

How many Advocate already joined the list, how can we fine them?

Thanks.

**Comment by 2xJDubs** [comment]
All future Advocate applicants, please us this link for the application form.

https://docs.google.com/forms/d/e/1FAIpQLSeuLOmRWNsXjaO8rt29hua44PVyETt0cpCH_7_5Y57GCQzflA/viewform?usp=sf_link
